Modify readme.md to explain how to install project (#9)
EmilieSchott/P8-ToDoCo: README.md

# An application to make Todo list

## Installation

---

1.  Clone this repository.

> 2.  With a CLI, place you in the project folder and run "composer install" command.

Reviewer: ```suggestion
2.  With a CLI, place yourself in the project folder and run "composer install".
```

Author: Oops... Thanks!

> 3.  In .env, choose your SGBD by remove the # before the DATABASE_URL required and complete asked informations. Then, type in the CLI the 2 following commands :

Reviewer: Keep the language consistent, "SGBD" is a French acronym ^^

```suggestion
3.  In .env, choose your DBMS by remove the # before the DATABASE_URL required and complete asked informations. Then, type in the CLI the 2 following commands :
```

Author: Yes, of course, sorry

    "php bin/console doctrine:database:create"

>     "php bin/console doctrine:schema:create"

Reviewer: If I'm not mistaken, the command is wrong:
```suggestion
    "php bin/console doctrine:schema:update --force"
```

Author: The command seems to work properly : I use it to install the initial project.
